reference/docs-conceptual/PowerShell-Core-Support.md

```diff
@@ -0,0 +1,69 @@
+# PowerShell Core Support Lifecycle
+
+PowerShell Core is a distinct set of tools and components that ships separately from Windows PowerShell.
+Therefore, PowerShell Core is not included in the Windows 10 or Windows Server licensing agreements.
+
+However, PowerShell Core is supported under traditional Microsoft support agreements, including [Premier][] and [Microsoft Enterprise Agreements][enterprise-agreement].
+You can also pay for [assisted support][] for PowerShell Core by filing a support request for your problem.
+
+We also offer [community support][] on GitHub where you can file an issue, bug, or feature request.
+We offer no guarantee there that your issue will be addressed or resolved in a timely manner.
+If you have a problem that requires immediate attention,
+you should use the traditional, paid support options.
+
+
+## Lifecycle of PowerShell Core
+
+PowerShell Core is adopting the [Microsoft Modern Lifecycle Policy][modern].
+This support lifecycle is intended to keep customers up-to-date with the latest versions.
+
+The version 6.x branch of PowerShell Core will be updated approximately once every six months (e.g. 6.0, 6.1, 6.2, etc.)
+
+> [!IMPORTANT]
+> You must update within six months after each new minor version release to continue receiving support.
+
+For example, if PowerShell Core 6.1 is released on July 1st, 2018,
+you would be expected to update to PowerShell Core 6.1 by January 1st, 2019 to maintain support.
+
+![PowerShell Core branch lifecycle][lifecycle-chart]
```

Comment:
`!` in front renders incorrectly still

Reply:
It will render incorrectly until he gets the image file linked. That is a TODO item.

Reply:
Ok, I didn't notice the `chart` previously.

Reply:
@joeyaiello  What happens if it gets discovered that a customer cannot upgrade due to a breaking change but the customer does not strictly need the upgrade. If the previous version was at least supported (in a limited fashion) then the customer could undo the upgrade and wait for the fix in the next release.
Does support mean that we'll end up seeing patches every month just because some enterprise customers like the ones of my company encounter blocking problems in special cases or would they get custom versions?

Reply:
@bergmeister I believe this is covered below by LTS although perhaps that section needs more details.  Once a LTS version is declared, let's say 6.5.0 as an example.  Then we would continue with 6.6.0, but also release 6.5.1 as needed on a servicing branch.  cc @joeyaiello